README.md

```diff
@@ -33,6 +33,10 @@ core distribution of the API and SDK.
 
 OpenTelemetry can collect tracing data using instrumentation. Vendors/Users can also create and use their own. Currently, OpenTelemetry supports automatic tracing for:
 
+### Erlang Instrumentation
```

Comment:
Should either have sub-headers in each of these for like `HTTP Server`, or have the main heading be like `HTTP Server` with subheadings under each for `Erlang` and `Elixir`.

Or don't put language at all since some like Cowboy could be used in both?

Reply:
Maybe we just ditch the language part for now and group by function.